hexrd/ui/interactive_template.py

```diff
@@ -41,6 +41,10 @@ def create_shape(self, module, file_name, det, instr):
         verts = self.panels['default'].cartToPixel(data)
         verts[:, [0, 1]] = verts[:, [1, 0]]
         self.shape = patches.Polygon(verts, fill=False, lw=1, color='cyan')
+        if np.any(np.isnan(verts)):
```

Comment:
FYI, there's a util function for this that's supposed to be the fastest way to check for nans in an array: `hexrd.ui.utils.has_nan`.

But this is probably fine if `verts` isn't going to be too big.

Reply:
Lets keep things consistent and simple since there is no guarantee we won't need a larger/more complicated boundary for a future instrument. Updated to use `hexrd.ui.utils.has_nan`. Thanks!